using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md

```diff
@@ -109,7 +136,54 @@ public void onNewToken(String token) {
 
 {% tab title="iOS (AEP 3.x)" %}
 
-Adobe Campaign Classic has not yet been released as an AEP 3.x Swift extension. Please reach out to your Adobe customer account manager if you have any questions or would like to express interest in the AEP 3.x Campaign Classic extension.
+{% hint style="info" %}
+To get your app ready to handle push notifications, see the tutorial on [configuring remote notification support](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/HandlingRemoteNotifications.html#//apple_ref/doc/uid/TP40008194-CH6-SW1). After you receive the Apple Push Notification service (APNs) token, send this token and the device information to Campaign Classic using the `registerDevice` API.
+{% endhint %}
+
+The `registerDevice` API registers a device with your Campaign Classic registration server. It takes the APNS token as a parameter with a user key that identifies a user, such as an email address or a login name. You can also provide a map of the custom key-value pairs that you want to associate with the registration. A boolean value is returned in the callback, which signals whether the registration was successful.
+
+### Swift
+
+**Syntax**
+
+```swift
+static func registerDevice(token: Data, userKey: String?, additionalParameters: [String: Any]?)
+```
+
+**Example**
+
+```swift
+func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
+  let params: [String: Any] = [
+    "name": "John",
+    "serial": 12345,
+    "premium": true
+  ]
+  CampaignClassic.registerDevice(token: deviceToken, userKey: "johnDoe", additionalParameters: params)
```

Comment:
Should we update the `userKey` value to be an email, same as 2.x example ("john@example.com")